website/docs/streaming-platforms/nifi.md

```diff
@@ -0,0 +1,205 @@
+---
+sidebar_position: 3
+sidebar_label: Apache NiFi on EKS
+---
+
+# Apache NiFi on EKS
```

Comment:
Please add some details on what Nifi is and when should customers consider Nifi. References to some pertinent documentation would also be useful here.

Reply:
Added intro section